processor/servicegraphprocessor/internal/store/edge.go

```diff
@@ -0,0 +1,63 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package store // import "github.com/open-telemetry/opentelemetry-collector-contrib/processor/servicegraphprocessor/internal/store"
+
+import "time"
+
+type ConnectionType string
+
+const (
+	Unknown         ConnectionType = ""
+	MessagingSystem ConnectionType = "messaging_system"
+	Database        ConnectionType = "database"
+)
+
+// Edge is an Edge between two nodes in the graph
+type Edge struct {
+	key string
+
+	TraceID                            string
```

Comment:
Are you using TraceID for any purposes? Perhaps use pcommon.TraceID instead of converting the []bytes to string?

Reply:
This was left from a previous implementation, removed!

Reply:
Changed to `pcommon.TraceID`.